include/sacn/cpp/source.h

```diff
@@ -843,7 +849,7 @@ inline std::vector<EtcPalMcastNetintId> Source::GetNetworkInterfaces(uint16_t un
 /**
  * @brief Get the current handle to the underlying C sacn_source.
  *
- * @return The handle or Source::kInvalidHandle.
+ * @return The handle, which may or may not be valid.
```

Comment:
I think we should be more specific: "The handle, which will only be valid if the source has been successfully created using Startup()"

Reply:
https://github.com/ETCLabs/sACN/pull/45/commits/6e80bb0110799e50953ba6a90aba3fe507716701